com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 ### Changed
 ### Fixed
+- Fixed a bug that caused out of project stream files to have the wrong number of material slots.
```

Comment:
@sebastienduverne this is previous merge's changelog entry.

Reply:
Suggestion:
`- Fixed a bug that caused files streamed from outside the project to have the wrong number of material slots.`